sources/packages/web/src/services/StudentService.ts

```diff
@@ -57,4 +58,8 @@ export class StudentService {
   async synchronizeFromUserInfo(): Promise<void> {
     return await ApiClient.Students.synchronizeFromUserInfo();
   }
+
+  async getProgramYears(): Promise<ProgramYear[] | null> {
```

Comment:
I do not see a reason to expect the return to be null in this case. We must ensure that there will be always a value for the program year, otherwise, just leave it thrown an exception, IMHO.